cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html

```diff
@@ -25,10 +25,17 @@
                            m-list__horizontal
                            m-global-eyebrow_languages">
                     <li class="m-list_item">
+                        {% if language == 'es' %}
```

Comment:
Would it be more accurate to specify `if language != 'en'` here? Since this isn't specific to Spanish?

Reply:
That's a good question. We do have a handful of pages in languages other than English or Spanish, but since they use the English header and footer, I'm not sure if it makes sense to show the English link for them.

Reply:
Ah okay that makes sense!